src/browser.ts

```diff
@@ -115,10 +139,16 @@ export class GrpcClient {
         metadata,
         callback
       ) => {
-        languageServiceStub[methodName].apply(languageServiceStub, [
-          req,
-          callback,
-        ]);
+        const cancelHandler = languageServiceStub[methodName].apply(
+          languageServiceStub,
+          [req, callback]
+        ) as CancelHandler;
+        return {
+          cancel: () => {
+            cancelHandler.canceller.abort();
+            cancelHandler.cancelRequested = true;
```

Comment:
Let's first set it to `true` to make sure that when it aborts the call, we are already prepared for that. It might work this way if `abort` is asynchronous but let's make our intention clear and put the assignment above the `.abort()`.

Reply:
Done.

Reply:
Still not here - did you push?

Reply:
Forgot to push that one on Friday, oops.